alpine/license.md

```diff
@@ -0,0 +1 @@
+View [license information](https://github.com/gliderlabs/docker-alpine/blob/master/LICENSE) for the software contained in this image.
```

Comment:
This is actually going to be multiple licenses for the major components of Alpine (musl and BusyBox). Is the license file required or should it just be removed?